dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -914,7 +914,14 @@ std::vector<BlobFileId> BlobStore::getGCStats()
             // Avoid divide by zero
             if (right_margin == 0)
             {
-                LOG_FMT_TRACE(log, "Current blob is empty [blob_id={}, total size(all invalid)={}].", stat->id, stat->sm_total_size);
+                assert(stat->sm_valid_rate == 0);
```

Comment:
Please don't use assert outside UT code. You can use `RUNTIME_ASSERT` or simply throw expcetions.